### DISCUSSION TITLE
Redux anti-pattern: state payload modified in action creator, not reducer

### DISCUSSION THREAD

**Discussion by danielweck** [edrlab/thorium-reader#1762] [Developer corner]
https://github.com/edrlab/thorium-reader/blob/58ef86ddb581ce37ba706f117ab986a9bbca2d30/src/main/redux/actions/win/registry/registerReaderPublication.ts#L23-L25

**Comment by danielweck** [comment]
See:

https://github.com/edrlab/thorium-reader/commit/544cf24a0a9c0e16ddabeeacec0561400e271b5c

**Comment by danielweck** [comment]
An issue worth keeping alive, but as a developer discussion until it is actionable more broadly in the codebase.

**Comment by panaC** [comment]
https://github.com/edrlab/thorium-reader/commit/478f16520fe08de12fd676cce7c8f59925cb7ce0
